Adds a "broken" diving suit for wrecks/corpses and visual adjustments

ubertpendragon opened the pull request:
Adds a "broken" diving suit from the unused broken diving suit sprite. 

-Uses the assests from Husked_DivingSuit.png to create a new wearable diving suit
The suit doesn't give any pressure resistance and will leak oxygen, can be deconstructed for lesser yields than a regular diving suit
![image](https://github.com/Regalis11/Barotrauma/assets/73229309/328ccae3-952d-4753-9a7e-89d842c426e8)

-Diving suit added to corpses loot as well as wrecked diving lockers
Some corpses will wear the suits, and the suit will spawn in wrecked diving suit lockers

![image](https://github.com/Regalis11/Barotrauma/assets/73229309/b09394e3-4a17-4d14-92f2-f64b964abbbe)

~~-Changed wrecked diving lockers sprite (see #12595)~~

---
# Sprite changes:

- ~~**MiscWrecked.png** adjusted the wrecked diving suit locker sprite to 75% opacity to be used as a decorativesprite~~

- **Headgears.png** color adjusted the broken helmet to match the suit tone and color. Also adjusts husks

- **Broken_Divingsuit.png** basically husked_divingsuit but with mild adjustments to shape and size to exactly match the regular diving suit model

- **Broken_DivingSuit_contained.png** assembled sprites for diving suit as an item; on the floor, contained, and inventory icon

Regalis11 commented:
Thank you for the PR! I'm not entirely sure of the reasoning behind these changes - is it to reduce the value of loot from the wrecks? Just for flavor (makes more sense for the suits to be degraded in wrecks)?

> MiscWrecked.png adjusted the wrecked diving suit locker sprite to 75% opacity to be used as a decorativesprite

I don't see why the texture needs to be adjusted for this - imo a better way would be to just draw the sprite with 75% opacity on the locker sprite (which would also mean the look of the cabinet doesn't change, since the base sprite is still the same). Also, for some reason this PR over doubles the file size of the atlas. The issue [has also already been fixed in our private repo](https://github.com/Regalis11/Barotrauma/issues/12595#issuecomment-1713348777).

ubertpendragon commented:
> Thank you for the PR! I'm not entirely sure of the reasoning behind these changes - is it to reduce the value of loot from the wrecks? To add more loot? Just for flavor (makes more sense for the suits to be degraded in wrecks)?

Mostly flavor, the sprite was unused, and it was weird seeing corpses dead without even trying to equip suits, or suits in lockers that are completely pristine

Balance wise this does dilute the "Loot Pool" for lockers, but also adds diving suits on the corpses themselves, which balances out.

> I don't see why the texture needs to be adjusted for this - imo a better way would be to just draw the sprite with 75% opacity on the locker sprite

If I knew it was possible I would've done that 😅 

> Also, it's practically impossible to see what has been changed from this PR, since the default files and the changes to them have been added in the same commit.

My bad.

I changed 2 things:
-The locker visuals (which I will remove)
-Added broken suits to Corpses inventory


All I did with corpses was sprinkle this on some existing ones:

```xml
      <Item identifier="brokensuit" equip="true">
        <Item identifier="oxygentank"/>
      </Item>
```

Feel to add to existing corpses or duplicate corpses (naked/suit) however you like


I'll update the PR to remove the locker stuff

Regalis11 commented:
Merged this to our private repo now (with some minor modifications), thank you for the contribution. :)
